cms/templates/home_page.html

```diff
@@ -19,16 +19,22 @@
 {% endblock %}
 
 {% block content %}
-  {% include "partials/hero.html" with action_title="Watch Now" action_url="#" %}
-  {% if page.upcoming_courseware %}
-    {% pageurl catalog_page as catalog_url %}
-    {% include "partials/course-carousel.html" with page=page.upcoming_courseware button_title="View All" button_url=catalog_url %}
-  {% endif %}
-  {% if page.about_mit_xpro %} {% include "partials/text-video-section.html" with page=page.about_mit_xpro %} {% endif %}
-  {% if page.learning_experience %} {% include "partials/learning-techniques.html" with page=page.learning_experience %} {% endif %}
-  {% if page.testimonials %} {% include "partials/testimonial-carousel.html" with page=page.testimonials %} {% endif %}
-  {% if page.inquiry_section %} {% include "partials/for-teams.html" with page=page.inquiry_section %} {% endif %}
-  {% if page.image_carousel_section %} {% include "partials/image-carousel.html" with page=page.image_carousel_section %} {% endif %}
+  {% with background_video_url=page.background_video_url upcoming_courseware=page.upcoming_courseware %}
+    {% with about_mit_xpro=page.about_mit_xpro learning_experience=page.learning_experience testimonials=page.testimonials %}
+      {% with inquiry_section=page.inquiry_section image_carousel_section=page.image_carousel_section %}
```

Comment:
These nested `with` blocks, along with the multiple assignments per expression, are not optimal for readability. Can we not just add these to the dictionary returned by `cms.models.CoursePage.get_context`, then reference them directly without needing to define these `with`s?